assets/shaders/basic-lighting.glsl

```diff
@@ -40,11 +40,11 @@ void main() {
 #pragma sokol @end
 
 #pragma sokol @fs fs
-uniform texture2D tex;
-uniform texture2D tex_emissive;
-uniform sampler smp;
+layout(binding=1) uniform texture2D tex;
```

Comment:
Do we need to explicitly set the layout now?

Reply:
in line 11 is `layout(binding=0)` for vertex shaders

then in this line you need to increase the number.

Reply:
The documentation says `Note that each resource type (uniform blocks, textures, samplers and storage buffers) has its own bindslot space which is shared across shader stages.` - since that fragment shader texture is the first texture, I believe it could be just bound to 0.

Reply:
You are totally right and now I understood it more, thanks! I updated all the shaders defs, ran shdc, updated sokol/graphics.zig and tested it is working.